bodhi/tests/server/functional/test_generic.py

```diff
@@ -101,6 +103,81 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    def test_home_counts(self):
+        user2 = User(name=u'dudemcpants')
+        self.db.flush()
+        self.db.add(user2)
```

Comment:
I don't think you need a user for this test since the home page should work unauthenticated, so you can probably drop these three lines.